### DISCUSSION TITLE
How to stop the prompt asking to allow sharing usage data

### DISCUSSION THREAD

**Discussion by Kreakdude** [felddy/foundryvtt-docker#940] [Q&A] [answered]
Every time I restart the stack or container I get the popup asking if I want to allow sharing usage data. It also says it will only ask it once. I guess it's supposed to persist that data in options.json but I'm guessing whatever is written to that file is lost unless it's in my docker compose. Is there an environment variable for this I missed?

I had a similar problem with the license agreement but I was able to solve it with this Q&A
https://github.com/felddy/foundryvtt-docker/discussions/159

Thanks if you can help.

**Comment by felddy** [comment, marked as answer]
Set the `FOUNDRY_TELEMETRY` environment variable to `true` or `false` to enable or disable the sharing of usage data.  

See: 
- https://github.com/felddy/foundryvtt-docker?tab=readme-ov-file#optional-variables
